dbt_server/services/filesystem_service.py

```diff
@@ -99,7 +120,7 @@ def write_unparsed_manifest_to_disk(state_id, previous_state_id, filedict):
     if previous_state_id and state_id != previous_state_id:
         #  TODO: The target folder is usually created during command runs and won't exist on push/parse
         #  of a new state. It can also be named by env var or flag -- hardcoding as this will change
-        #  with the click API work
+        #  with the click API work. This bypasses other use DBT_TARGET_PATH
```

Comment:
Question/nit: should this read "This bypasses other uses of DBT_TARGET_PATH"?

Reply:
Not entirely sure what I was going for, thanks for the catch 😅